src/components/Packages/PackageList.js

```diff
@@ -5,11 +5,18 @@ import {Row, Col} from 'reactstrap';
 class PackageList extends Component {
 
     componentDidMount() {
-        this.props.getPackages();
+        if(!this.isRemoteDataLoaded()) {
```

Comment:
I understand what you did here...I am trying to understand why.

I am a little concerned that we might get into a loop here if there is an error with getting packages.